addons/popochiu/engine/templates/hotspot_template.gd

```diff
@@ -17,6 +17,14 @@ func _on_click() -> void:
 #	await C.player.say("What a nice view")
 
 
+func _on_double_click() -> void:
+	# Replace the call to E.command_fallback() with your code.
+	E.command_fallback()
+	# For example, you could make the player instantly goto another room on an exit hotspot instead
+	# of waiting for the player to walk there.
+#	await E.goto_room("NewRoom")
+
+
```

Comment:
The `_on_double_click()` virtual method should be included in the templates for props and characters too.